pkg/sriov/sriov.go

```diff
@@ -127,10 +127,6 @@ func (s *sriovManager) ReleaseVF(conf *sriovtypes.NetConf, podifName string, net
 		return fmt.Errorf("failed to get init netns: %v", err)
 	}
 
-	if len(conf.ContIFNames) < 1 && len(conf.ContIFNames) != len(conf.OrigVfState.HostIFName) {
```

Comment:
Should we keep a check like `conf.ContIFName != conf.OrigVfState.HostIFName` ?

Reply:
it's ok to have a different name for the interface in the container. In fact, I expect that to be the most common case.